### DISCUSSION TITLE
Lession 6: get entrance fee test error

### DISCUSSION THREAD

**Discussion by sl00k** [smartcontractkit/full-blockchain-solidity-course-py#414] [Q&A]
![image](https://user-images.githubusercontent.com/33859801/142763227-a2cb00f4-d71f-446d-8211-5e62ca593254.png)

I can not run the test. But how do I actually find out where the error is here? I set the prices to 0 and 5 to test against to get sure the entrance fee is in that limit.

**Comment by cromewar** [comment]
Hello @sl00k the transaction is most likely failing due to for "eth_usd_fee", don't forget to deploy de mocks your the price feed on deploy, and also take in mind your entry free is going to be 50$ so use [this](https://coinmarketcap.com/es/converter/eth/usd/) to calc the equivalence. If nothing works please post here your code so we can check it.
